packages/wonder-blocks-modal/src/components/modal-dialog.js

```diff
@@ -94,9 +86,8 @@ export default class ModalDialog extends React.Component<Props> {
                             {below && <View style={styles.below}>{below}</View>}
                             <View
                                 role={role}
+                                {...ariaProps}
```

Comment:
**suggestion:** It would be better to only pass what is needed. There are a lot of ARIA attributes that we surely won't need here.